dask/dataframe/io/parquet.py

```diff
@@ -74,6 +74,11 @@ def _read_fastparquet(fs, paths, myopen, columns=None, filters=None,
     elif len(minmax) > 1:
         if index:
             index_col = index
+        elif index is None:
+            # attempt auto-index
+            index = pf._get_index()
+            if index:
+                index_col = index
         elif 'index' in minmax:
             index_col = 'index'
```

Comment:
We do not have the metadata for files not written from pandas/dask. In that case we still want to use an appropriate index, if possible.